### PR TITLE
misc: generate snapshot/timespan reports from sample flow result

### LINKED ISSUE #13935: `update:snapshot-sample-json` should be kept in sync with master
The other samples jsons (normal report, flow report) are verified to have no changes in CI every commit, but not this one.

### PR DESCRIPTION
**Pull request by adamraine**
Closes #13935

### CODE REVIEW

**Comment by brendankenny** [comment]
line count change, +5, -2508

🥲

**Comment by adamraine** [comment]
I'm paying for the sins I committed in https://github.com/GoogleChrome/lighthouse/pull/13715

File: package.json [hunk 1]
```diff
@@ -74,7 +74,6 @@
     "i18n:collect-strings": "node lighthouse-core/scripts/i18n/collect-strings.js",
     "update:lantern-baseline": "node lighthouse-core/scripts/lantern/update-baseline-lantern-values.js",
     "update:sample-artifacts": "node lighthouse-core/scripts/update-report-fixtures.js",
-    "update:sample-json": "yarn i18n:collect-strings && node ./lighthouse-cli -A=./lighthouse-core/test/results/artifacts --config-path=./lighthouse-core/test/results/sample-config.js --output=json --output-path=./lighthouse-core/test/results/sample_v2.json && node lighthouse-core/scripts/cleanup-LHR-for-diff.js ./lighthouse-core/test/results/sample_v2.json --only-remove-timing",
```
Comment (connorjclark) [inline package.json:77]: uhhhhh

Comment (adamraine) [reply package.json:77]: Lol deleted the wrong one

**Review by connorjclark — APPROVED** [review]

**Pull request merged**